app/bundles/PluginBundle/Assets/js/plugin.js

```diff
@@ -1,7 +1,18 @@
 /* PluginBundle */
 Mautic.matchedFields = function (index, object, integration) {
+    var compoundMauticFields = ['mauticContactTimelineLink', 'mauticContactIsContactableByEmail'];
```

Comment:
I think it's better to avoid defining the fields on 2 places in PHP and in JS. To avoid that, could we simply check if the field has the `disabled` attr which is what PHP does and act based on that?

Reply:
Not really because that attribute is only set when it's being edited, if we are assigning the field for the first time, there is no way of knowing.